plugin/src/test/groovy/de/gesellix/gradle/docker/tasks/DockerDisposeContainerTaskSpec.groovy

```diff
@@ -67,4 +67,16 @@ class DockerDisposeContainerTaskSpec extends Specification {
         then:
         0 * dockerClient._
     }
+
+    def "allows to removeVolumes"() {
```

Comment:
Would you mind adding another test case showing the default `v: 0`?

Reply:
As I had to fix the other tests anyways, is that sufficient for you, or do you want a separate test for that additionally?

Reply:
I'd prefer to have a dedicated test as kind of documentation for "expected behaviour", because the official Docker CLI might have another default (`v: nil` instead of `v: 0`).